recon_wizard_template.py

```diff
@@ -60,14 +61,26 @@ def update_recon_configuration_step_state():
         st.session_state[UPDATE_RECON_KEY] = True
 
     if st.session_state[UPDATE_RECON_KEY]:
-        recon_function_string = get_recon_script(RECON_NAME)
+        original_analysis = MitoAnalysis.from_json(get_recon_analysis(RECON_NAME))
 
-        recon_function_string, original_imported_df_names = remove_import_code_and_get_df_names(recon_function_string)
+        def map_param_to_name(param: ParamMetadata):
+            return param['name']
```

Comment:
Rather than getting the param name, which results in 
```Import a file to replace file_name_import_csv_0 dataframe.``` (not super informative)

Let's get the file value? And then strip the file so we only have the actual final file name (not the full path). That way, it's really informative what you need to upload again.